aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -203,11 +244,18 @@ class Route:
     def __init__(
         self,
         method: str,
+        path: str,
```

Comment:
We now store the raw path that was supplied by the user, which will be useful later when we need to generate the OpenAPI spec